packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1627,6 +1627,19 @@ async function handler(
     })
   );
 
+  // update original rescheduled booking (no seats event)
+  if (!eventType.seatsPerTimeSlot && originalRescheduledBooking?.uid) {
+    await prisma.booking.update({
```

Comment:
The update unsets the idempotencyKey - so it needs to happen before createBooking to prevent a conflict.